src/modules/oembed-vm.js

```diff
@@ -21,62 +21,77 @@
 			abstractProvider = p;
 		});
 		var vimeoProvider = $.extend({}, abstractProvider, {
+			options: {},
 			getIframe: function (id, autoplay, loop, rel, extra) {
-				autoplay = autoplay !== undefined ? autoplay : 1;
-				loop = loop !== undefined ? loop : 1;
-				
-				return abstractProvider.getIframe()
-					.attr('src', '//player.vimeo.com/video/' + id +
-							'?autoplay=' + autoplay + '&loop=' + loop +
-							'&muted=' + autoplay +
-							'&player_id=' + 'vimeoPlayer' + id +
-							'&api=1&html5=1&rel=' + rel + (extra || ''));
+				vimeoProvider.options[id].autoplay = autoplay ? (autoplay === 1) : true;
+				vimeoProvider.options[id].loop = !isNaN(loop) ? (loop === 1) : true;
+				vimeoProvider.options[id].rel = rel;
+				// No need to create the iframe: vimeo's SDK is going to do it
+				return null;
 			},
 			
 			ready: function (container, callback) {
-				App.loaded($f, function ($f) {
-					var player = $f($('iframe', container).get(0));
-					
-					player.element.id = 'vimeoPlayer' + container.attr('data-oembed-id');
+				var id = container.attr('data-video-id');
+				var options = vimeoProvider.options[id];
+				App.loaded(VM, function (Player) {
+					var player = new Player(container.get(0), {
+						id: id,
+						background: false,
+						autopause: true,
+						autoplay: options.autoplay,
+						byline: false,
+						playsinline: true,
+						controls: true,
+						loop: options.loop,
```

Comment:
They should be settable by data attributes too.

Reply:
and fallback to the options.* settings.

Reply:
Yes. This is out of scope tho.

Reply:
You are more than welcome to send another pr then ;) haha